app/main.py

```diff
@@ -1,8 +1,55 @@
 class Car:
-    # write your code here
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+
     pass
+cars = [Car]
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power= clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+
+    def serve_cars(self, cars: list) -> list:
+        served_cars = []
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                served_cars.append(car)
+        # for car in served_cars:
+        #     car.clean_mark = self.clean_power
```

Comment:
These lines are commented out but seem to be intended to update the `clean_mark` of each car in `served_cars`. Ensure that this logic is handled elsewhere if these lines are not needed.